source/slang/slang-type-layout.cpp

```diff
@@ -1817,6 +1817,7 @@ LayoutRulesFamilyImpl* getDefaultLayoutRulesFamilyForTarget(TargetRequest* targe
     case CodeGenTarget::ShaderSharedLibrary:
     case CodeGenTarget::CPPSource:
     case CodeGenTarget::CSource:
+    case CodeGenTarget::WGSL:        
```

Comment:
This may not be correct, I think you want wgl to have similar layout rule with GLSL/SPIRV case for now.

Reply:
If there are actual difference in type alignment and layout rules compared to spirv, we will need to branch out from that and have dedicated type layout classes for WGSL, but I think moving this line to after 1810 is a good start.